src/specs/index.js

```diff
@@ -6,6 +6,7 @@ describe('ui-harness', function() {
   require('./it-server.spec');
   require('./PropTypes.spec');
   require('./page.spec');
+  require('./typescript.spec.tsx')
```

Comment:
Yeah you actually can, this was just a result of me trying to debug that paths issue. You definitely can just have `./typescript.spec`